src/global.css

```
@@ -19,6 +19,17 @@ body {
   height: var(--navbar-height);
 }
 
+nav {
```

It's possible to have multiple `<nav>` elements on a page, right? Should this be more specific to the top nav? For our demo application, it likely will never matter. But would consider it a bit safer to do the following:

```suggestion
nav.topnav {
```